test/helper_test.rb

```diff
@@ -1,39 +1,44 @@
 require "test_helper"
 
 class HelperTest < ActionView::TestCase
+  tests Webpacker::Helper
+
+  attr_reader :request
+
   def setup
-    @view = ActionView::Base.new
-    @view.extend Webpacker::Helper
+    @request = Class.new do
+      def send_early_hints(links) end
+    end.new
   end
 
   def test_asset_pack_path
-    assert_equal @view.asset_pack_path("bootstrap.js"), "/packs/bootstrap-300631c4f0e0f9c865bc.js"
-    assert_equal @view.asset_pack_path("bootstrap.css"), "/packs/bootstrap-c38deda30895059837cf.css"
+    assert_equal "/packs/bootstrap-300631c4f0e0f9c865bc.js", asset_pack_path("bootstrap.js")
+    assert_equal "/packs/bootstrap-c38deda30895059837cf.css", asset_pack_path("bootstrap.css")
   end
 
   def test_javascript_pack_tag
     assert_equal \
       %(<script src="/packs/bootstrap-300631c4f0e0f9c865bc.js"></script>),
-      @view.javascript_pack_tag("bootstrap.js")
+      javascript_pack_tag("bootstrap.js")
```

Comment:
Ahh, looks like it's written the opposite way everywhere

Reply:
Oh, My understanding is... "expected" is left position, and "actual" is right position.

https://github.com/seattlerb/minitest/blob/master/lib/minitest/assertions.rb#L159-L188

But If I need to modify all test cases, I'll do it. I'll respect to your opinion :)

Reply:
Ahh, yes that's right 👍  kinda of confusing though, depending on how one interprets `expected` and `actual`